test/test_conffile.py

```diff
@@ -24,13 +24,33 @@ def setUp(self):
 No really.
 """)
 
+    def tearDown(self):
+        try:
```

Comment:
Could you use shutil.rmtree() here? That may simplify the code.

Reply:
Thanks, fixed that.